src/assistants/openai/openai_assistant.rs

```diff
@@ -83,10 +80,12 @@ impl OpenAIAssistant {
      */
     async fn create_assistant(&mut self) -> Result<()> {
         //Get the assistant API url
-        let assistant_url = format!("{}/assistants", self.version.get_endpoint());
+        let assistant_url = self
```

Comment:
I actually think this is now a much more elegant way of building urls that hardcoding paths.

Reply:
agreed!!